### PR TITLE
MGMT-4813: If not provided, use INSTALLER_KUBECONFIG env

### PR DESCRIPTION
**Pull request by RazRegev**
in order to determine to which cluster we deploy CRDs to,
in needs to be a separate env than KUBECONFIG which uses
to represents the installed cluster kubeconfig path.

### CODE REVIEW

File: discovery-infra/tests/conftest.py [hunk 1]
```diff
@@ -54,6 +54,7 @@ def _get_cluster_name():
                  "cluster_name": _get_cluster_name(),
                  "private_ssh_key_path": utils.get_env('PRIVATE_KEY_PATH', private_ssh_key_path_default),
                  "kubeconfig_path": utils.get_env('KUBECONFIG', ''),
+                 "installer_kubeconfig_path": utils.get_env('INSTALLER_KUBECONFIG', None),
```
Comment (osherdp) [inline discovery-infra/tests/conftest.py:57]: we need to add ``INSTALLER_KUBECONFIG`` to ``skipper.env``, no?
also giving it a default value on the ``Makefile``

**Review by osherdp — CHANGES_REQUESTED** [review]

File: discovery-infra/tests/conftest.py (hunk 1 above)
Comment (RazRegev) [reply discovery-infra/tests/conftest.py:57]: added to skipper.env 
not sure we need to add it to Makefile, if not set it will go to the default path

**Pull request merged**